sql/builtin_functions.cc

```diff
@@ -3141,6 +3141,35 @@ BoundFunction hash(const std::vector<BoundSqlExpression> & args)
 
 static RegisterBuiltin registerHashFunction(hash, "hash");
 
+BoundFunction tryFct(const std::vector<BoundSqlExpression> & args)
+{
+    checkArgsSize(args.size(), 2);
```

Comment:
You can do a separate implementation for if we have one or two args based upon args.size() here.